apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -99,18 +272,26 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
   end
 
   @tag fixtures: [:processor_empty, :processor_filled]
-  test "can process empty new exits or empty finalizations", %{processor_empty: empty, processor_filled: filled} do
+  test "can process empty new exits, empty in flight exits or empty finalizations", %{
+    processor_empty: empty,
+    processor_filled: filled
+  } do
     assert {^empty, []} = Core.new_exits(empty, [], [])
+    assert {^empty, []} = Core.new_in_flight_exits(empty, [], [])
     assert {^filled, []} = Core.new_exits(filled, [], [])
+    assert {^filled, []} = Core.new_in_flight_exits(filled, [], [])
+
     assert {^filled, []} = Core.finalize_exits(filled, {[], []})
   end
 
-  @tag fixtures: [:processor_empty, :alice, :state_empty, :events]
+  @tag fixtures: [:processor_empty, :alice, :state_empty, :exit_events]
   test "handles invalid exit finalization - doesn't forget and activates", %{
```

Comment:
"and activates"? Please elaborate.

Reply:
"and activates" => "and causes a byzantine chain report"